packages/quantic/cypress/e2e/default-2/generatedAnswer/generated-answer-selectors.ts

```diff
@@ -60,10 +60,6 @@ export const GeneratedAnswerSelectors: GeneratedAnswerSelector = {
     GeneratedAnswerSelectors.get()
       .find('[data-cy="generated-answer__citations"] .citation__title')
       .eq(index),
-  citationIndex: (index: number) =>
```

Comment:
We probably also have expectations in the `generated-answer-expectaons.ts` file that use this, so we should clean them out as well.

Reply:
we do! 👍

Reply:
oh my ctrl+f didnt find it :D